setup.py

```diff
@@ -97,7 +97,7 @@
     name="pyautogen",
     version=__version__,
     author="AutoGen",
-    author_email="auto-gen@outlook.com",
+    author_email="autogen-contact@service.microsoft.com",
```

Comment:
Thank you @ekzhu , @gagb and @qingyun-wu .  Would you like to add all the non-MS maintainers to the distribution list?

Reply:
@thinkall, @qingyun-wu who is a non-MS maintainer is already on the list.

Reply:
Thank you @gagb , are there any other non-MS maintainers or collaborators you think  worth adding to the list?

Reply:
That's a good question. Can you send suggestions of names? Also the see PR https://github.com/microsoft/autogen/pull/3384 that adds CONTRIBUTORS.md instead of external Notion-based link.